packages/reason/reason.1.4.0/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "1.2"
+maintainer: "Jordan Walke <jordojw@gmail.com>"
+authors: "Jordan Walke <jordojw@gmail.com>"
+homepage: "https://github.com/facebook/reason"
+bug-reports: "https://github.com/facebook/reason/issues"
+license: "BSD"
+doc: "http://facebook.github.io/reason"
+tags: "syntax"
+dev-repo: "git://github.com/facebook/reason.git"
+substs: "pkg/META"
+build: [
+  [make "compile_error"]
+  [
+    "ocaml"
+    "pkg/build.ml"
+    "native=%{ocaml-native}%"
+    "native-dynlink=%{ocaml-native-dynlink}%"
+    "utop=%{utop:installed}%"
+  ]
+]
+build-test: [
+  "ocamlbuild"
+  "-classic-display"
+  "-use-ocamlfind"
+  "src_test/test_reason.byte"
+  "--"
+]
+depends: [
+  "easy-format" {>= "1.2.0"}
+  "ocamlfind" {build}
+  "utop" {>= "1.17"}
+  "BetterErrors" {>= "0.0.1"}
+  "menhir" {>= "20160303"}
+  "merlin" {>= "2.5.0"}
+  "merlin-extend" {>= "0.3"}
+  "re" {>= "1.5.0"}
+]
+conflicts: [
+  "utop" {< "1.17"}
```

Comment:
This `conflicts` constraint isn't strictly necessary, since there's an equivalent constraint on the `utop` `depends` entry.